code/game/gamemodes/miniantags/slaughter/bloodcrawl.dm

```diff
@@ -68,13 +68,16 @@
 			for(var/i in 1 to 3)
 				playsound(get_turf(src), sound, 100, 1)
 				sleep(30)
-			if(kidnapped)
-				to_chat(src, "<B>You devour [kidnapped]. Your health is fully restored.</B>")
-				adjustBruteLoss(-1000)
-				adjustFireLoss(-1000)
-				adjustOxyLoss(-1000)
-				adjustToxLoss(-1000)
 
+			if(kidnapped)
+				if(ishuman(kidnapped) || isrobot(kidnapped))
+					to_chat(src, "<B>You devour [kidnapped]. Your health is fully restored.</B>")
+					adjustBruteLoss(-1000)
+					adjustFireLoss(-1000)
+					adjustOxyLoss(-1000)
+					adjustToxLoss(-1000)
+				else
+					to_chat(src, "<B>You devour [kidnapped], but this measly meal does not sate your appetite.")
```

Comment:
this is leftover oldcode but this should really use a <span> with a defined class instead of an unclosed <b> bold element.

Reply:
I suggest using warning here:

```suggestion
					to_chat(src, "<span class='warning'>You devour [kidnapped], but this measly meal does not sate your appetite!</span>")
```